internal/manifest/commit_deployment.go

```diff
@@ -160,6 +160,7 @@ func (p *OSTreeDeployment) serialize() osbuild.Pipeline {
 			Keymap: p.Keyboard,
 		}
 		keymapStage := osbuild.NewKeymapStage(options)
+		keymapStage.MountOSTree(p.osName, p.osTreeRef, 0)
```

Comment:
Yes exactly.  When we're building the commit, we built it like a normal tree then convert.  But now we need to operate on an existing deployment, so we need to mount `/etc` and friends.

In the past, stages like `fstab` had special options for this, but a while ago, osbuild got the ostree mount type so that any stage can be applied to an ostree deployment.

Reply:
Thanks for the confirmation and additional context. The new approach looks much nicer IMHO.